pipelines/enforce-github-secrets-trigger.rego

```diff
@@ -0,0 +1,22 @@
+################################################
+# Persistence Policies
+
+## pakcage declaration for PE Plugin
+package spinnaker.execution.pipelines.before
+
+## pakcage declaration for Armory Extension
+# package opa.pipelines
+
+deny["Every pipeline github trigger must have a secret"] {
+	some i
+    trigger := input.triggers[i]
+	trigger.type == "git"
+    trigger.source == "github"
+    secret := object.get(trigger, "secret", "")
+    secret == ""
+}
+
+response := {
+	"allowed": count(deny) == 0,
+	"errors": deny,
+}
```

Comment:
Isn't `response` only necessary if we're using the DAS? Do you think it makes sense to include it here if the majority of users won't be using the DAS?

Reply:
Thats a good question. We don't need to return it, but I think it would be nice to standardize how the api works. Just my thoughts.